unit-tests/post-processing/test-post-processing.py

```diff
@@ -0,0 +1,100 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import test
+from threading import Thread
+import tempfile
+import os.path
+import numpy as np
+
+################################################################################################
+test.start("Unit transform test")
+ctx = rs.context()
+
+temp_dir = tempfile.gettempdir()
+log_filename = os.path.join(temp_dir, 'port_legacy.log')
+rs.log_to_file(rs.log_severity.debug, 'port_legacy.log')
```

Comment:
Why do we need to log to a file?
Maybe those 3 lines are not necessary?

Reply:
removed